src/components/Loans/OpenLoans/components/OpenLoansWithStaticData/OpenLoansWithStaticData.js

```diff
@@ -13,6 +13,9 @@ import {
 import { stripesShape } from '@folio/stripes/core';
 import { NoValue } from '@folio/stripes/components';
 
+import {
+  getLoanLastReminderNumber,
```

Comment:
This function was created in:

https://github.com/folio-org/ui-users/pull/2510/files#diff-b0d4a58329d2ccc2283d1d9646c187d2a9a18618ced2266c9f647f5f081f35c0R13